### PR TITLE
Minor changes

### LINKED ISSUE #11: Be specific about Base64 encoding - needs to be Base64 URL encoding

### LINKED ISSUE #18: Bi-directional statement in abstract
In the abstract of the current spec, the following statement is made 

```
The DID Configuration resource provides proof of a bi-directional relationship between the controller of an Internet domain and a DID via cryptographically verifiable signatures that are linked to a DID's key material
```

Can we clarify further what is meant by this? This implies you can start with a did and discover associated domains from it? That may just be my interpretation but to be clear, the current design does not allow this?

### PR DESCRIPTION
**Pull request by tplooker**
- Changes the referenced did spec to the WG version
- Attempts to add clarifying language around discovery of the well-known resource, fixes #18 
- Changes the language to refer to compact serialization about the JWT, fixes #11
- Adds clarity to interpretation of the domain field in the JWT
- Adds further clarity to DID resolution and guidance about what key material must be used from the DID document to perform the signature validation correctly

### CODE REVIEW

**Comment by tplooker** [comment]
@OR13 @csuwildcat any thoughts on this PR? Do you feel it addresses the referenced issues?

File: resources/did-configuration/index.html [hunk 1]
```diff
@@ -217,7 +223,7 @@ <h2>Validation of Domain Linkage Assertions</h2>
...
+            If the resource processing entity is not looking for the presence of a specific DID, or intending to validate a specific DID in the `entries` array of Domain Linkage Assertions, the resource processing entity SHOULD iterate the array of Domain Linkage Assertions beginning at 0 index. This normative iteration from 0 index is intended to allow domain controllers to order their Domain Linkage Assertions in accordance with a known processing order, in cases where order may matter to the domain controller. An example of this the case where a domain controller has a primary signing DID that is the likely target of interest for the vast majority of relying parties who seek to validate an assertion based on the DIDs associated with a domain.
```
Comment (OR13) [inline resources/did-configuration/index.html]: spelling error "An example of this the case where a domain controller has a primary...."

**Comment by OR13** [comment]
I like these changes, especially updating the links, and adding the section regarding authentication.

**Comment by tplooker** [comment]
@OR13 issue fixed, do you think the location of the section about discovery is correct or should it feature elsewhere?

**Comment by OR13** [comment]
@tplooker I think its good where it is, please add yourself to the contributors list in this PR :)

**Comment by tplooker** [comment]
Done, thanks.

**Pull request merged**